nativescript-charter.md

```diff
@@ -0,0 +1,125 @@
+# NativeScript Charter
+
+## Section 0. Guiding Principle
+
+NativeScript is part of the OpenJS Foundation. The project operates transparently, openly, collaboratively, and ethically. Project proposals, timelines, and status must not merely be open, but also easily visible to outsiders.
+
+## Section 1. Scope
+
+NativeScript provides platform APIs directly to the JavaScript runtime (with strong types) for a rich TypeScript development experience. As an open-source framework to develop apps for iOS, visionOS, Android, macOS and other platforms combining a best of all worlds approach marrying familiar Web approaches like CSS and view templating with common platform languages (Swift, Kotlin, Objective-C, Java) it delivers a liberating toolset for developers.
+
+It combines a node based CLI as an approachable interaction point for JavaScript developers to interact with native platform development spanning Xcode and Android Studio processes.
```

Comment:
Since this document will be difficult to change, it might be better to leave this more genric, e.g. 
```It combines a CLI as an approachable interaction point for JavaScript developers to interact with native platform development processes.```